Documentation.md

```diff
@@ -483,42 +483,39 @@ Subscribe a method to be invoked whenever the contact is updated.
 ```
 contact.onIncoming(function(contact) { ... });
 ```
-Subscribe a method to be invoked when the contact is incoming.  In this state, the contact is waiting to be
+Subscribe a method to be invoked when a queue callback contact is incoming.  In this state, the contact is waiting to be
 accepted if it is a softphone call or is waiting for the agent to answer if it is not a softphone call.
 
 ### `contact.onPending()`
 ```
 contact.onPending(function (contact) { ... });
 ```
-Subscribe a method to be invoked when the contact is pending. Pending occurs after incoming and before connecting, when an agent has accepted or answered, but the backend has not yet acknowledged that a connection is being formed.
+Subscribe a method to be invoked when the contact is pending. This event is expected to occur before the connecting event.
 
 ### `contact.onConnecting()`
 ```
 contact.onConnecting(function (contact) { ... });
 ```
-Subscribe a method to be invoked when the contact is connecting. This works with chat and softphone contacts. This state happens after the agent has accepted a softphone call or answered if not. 
+Subscribe a method to be invoked when the contact is connecting. This works with chat and softphone contacts. This event happens when a call or chat comes in, before accepting (there is an exception for queue callbacks, in which onConnecting's handler is executed happens after the callback is accepted). Note that once the contact has been accepted, the `onAccepted` handler will be triggered.
```

Comment:
I think there's a small typo here, should be "in which onConnecting's handler is executed after", not "happens after".